app/schema.json

```diff
@@ -209,14 +212,23 @@
               "children": { "type": "array" }
             }
           },
+          "created": { "type": "string" },
           "server": { "type": "string" },
           "tilesetUid": { "type": "string" },
           "position": { "type": "string" },
           "options": { "type": "object" },
           "name": { "type": "string" },
           "transforms": { "type": "array" },
           "width": { "type": "number" },
-          "height": { "type": "number" }
+          "height": { "type": "number" },
+          "binsPerDimension": { "type": "number" },
```

Comment:
This is returned by the server in tilesetInfo and shouldn't be here. It's erroneously included in exported viewconfs and viewconfs that contain it are incorrect.

Reply:
Same question: I lean towards correcting these viewconf fixtures.